backend/endpoints/order.py

```diff
@@ -0,0 +1,163 @@
+# order.py
+
+from datetime import datetime
+from flask import Blueprint, request, jsonify, current_app
+from models import db, Customer, Package, Order, OrderDetails, DeliveryTypeEnum
+import json
+from endpoints.auth import get_current_user
+
+order = Blueprint('order', __name__)
+
+CURRENT_USER_FILE = 'current_user.json'
+
+@order.route("/create_order", methods=["POST"])
+def createOrder():
```

Comment:
to follow the nomenclature, function names should be with an underscore and not camelCase